Automattic/newspack-theme
# Chore: update language file lines; add updated string for Spanish

### All Submissions:


### Changes proposed in this Pull Request:

This PR updates the string numbers for the rest of the theme translations, to match the updates in #1260.

It also removes a couple 'fuzzy' translations from the Spanish translation (bad attempts at matches on PoEdit's part); renames the new .JSON file to be prefixed with `newspack-`, and makes it looks like I made other changes to the Spanish JSON files, even though I think it was just something weird I correcting conflicts 😳 

### How to test the changes in this Pull Request:

1. Apply the PR.
2. Cycle through the translations and confirm they're still applied on the front-end, and in the editor (the best spot to check there is the 'Featured Image' options, since 'Large' should be translated in all of the languages. The theme includes translations for Spanish, German, Portuguese, Brazilian Portuguese and Belgian French. 
 
### Other information:
